warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/EventMapper.java

```diff
@@ -100,11 +102,8 @@ public class EventMapper<K1,V1 extends RawRecordContainer,K2,V2> extends StatsDE
 
     private static final Logger log = Logger.getLogger(EventMapper.class);
 
-    /**
-     * number which will be used to evaluate whether or not an Event should be processed. If the Event.getEventDate() is greater than (now - interval) then it
-     * will be processed.
-     */
-    public static final String DISCARD_INTERVAL = "event.discard.interval";
+    public static final String DISCARD_INTERVAL = DataTypeDiscardIntervalPredicate.DISCARD_INTERVAL;
```

Comment:
Thanks for moving out the discard interval predicate.  It looks like this line can be removed.

Reply:
See suggested edit below.

Reply:
Actually there are other classes that are looking for the constant in the EventMapper, so I was trying to minimize those changes.

Reply:
Well maybe its only in one place, so I will do it.